src/HonzaBotner.Discord.Services/Options/InfoOptions.cs

```diff
@@ -5,8 +5,7 @@ namespace HonzaBotner.Discord.Services.Options
     public class InfoOptions
     {
         public static string ConfigName => "InfoOptions";
-
-        public string Version { get; set; }
+        public string VersionSuffix { get; set; } = "";
```

Comment:
Do we really need to set it to an empty string? We will handle that anyway with programmatically added `-` change.